neqo-http3/src/connection_client.rs

```diff
@@ -2193,7 +2239,7 @@ mod tests {
         let e = client.events().next().unwrap();
         if let Http3ClientEvent::DataReadable { stream_id } = e {
             assert_eq!(stream_id, request_stream_id);
-            let mut buf = [0u8; 100];
+            let mut buf = [0_u8; 100];
```

Comment:
No need to change this now, but you probably only need to do this:

```suggestion
            let mut buf = [0; 100];
```